pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/DefaultSchemasTest.java

```diff
@@ -67,30 +61,6 @@ public void testReaderInstantiation() {
         Assert.assertNotNull(stringReaderBuilder);
     }
 
-    @Test
-    public void testStringSchema() throws Exception {
-        String testString = "hello world";
-        byte[] testBytes = testString.getBytes(StandardCharsets.UTF_8);
-        StringSchema stringSchema = new StringSchema();
-        Assert.assertTrue(stringSchema.decode(testBytes).equals(testString));
-        assertEquals(stringSchema.encode(testString), testBytes);
-
-        Message<String> msg1 = MessageBuilder.create(stringSchema)
```

Comment:
lets just get rid of the MessageBuilder portion  of the test since since that is V1 API  but we should keep the rest of the test.

Reply:
@jerrypeng Updated